nbgrader/_version.py

```diff
@@ -1,2 +1,2 @@
-version_info = (0, 8, 0, 'dev')
+version_info = (0, 8, 0)
```

Comment:
Probably we'll want to stick to what is on `master` currently? And let the releaser do the bump to the new version on release.

Reply:
I agree but I'm not sure how to handle this using `tbump`. 
`tbump` uses python string format() to patch version, and complains if I try to change from `(0, 8, 0, "dev")` to `(0, 8, 0)` :

```
Error: nbgrader/_version.py: refusing to replace by version containing 'None'
More info:
 * version groups:  {'major': '0', 'minor': '8', 'patch': '0', 'channel': None}
 * template:        ({major}, {minor}, {patch}, "{channel}")
 * version:         (0, 8, 0, "None")
```

Does the releaser handle this ? Otherwise, it means that a *channel* must be included for future versions.

Reply:
You are right, I had missed the `field` option